### ISSUE TITLE
"i" is inserted into fzf buffer on neovim 0.4.0

### ISSUE THREAD

**Issue by expipiplus1** [autozimu/LanguageClient-neovim#793]
To reproduce:

- Call `LanguageClient#textDocument_codeAction()`
- Observe that there has been an `i` inserted in the `fzf` buffer

These lines seem to be to blame, removing them prevents the `i` being inserted: https://github.com/autozimu/LanguageClient-neovim/blob/dce2a8e11f1a246c25f30a7668f3ab3e9eb0043e/autoload/LanguageClient.vim#L213-L215

```
 $ nvim --version
NVIM v0.4.0-dev
Build type: Release
LuaJIT 2.1.0-beta3
Compilation:
Compiled by j

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "
/nix/store/pb6i2hbiyv83mcjp4f7vdx7mbd7k4rcf-neovim-unwrapped-0.4.0/share/nvim"

```

```
languageclient --version
languageclient 0.1.145
```

**Comment by autozimu** [comment]
Thanks for reporting.

This was added to workaround the issue fzf was started in normal mode in neovim. Seems like the issue was addressed in neovim 0.4.0 and the workaround is causing issue.

Will fix it in a compatible way shortly.

**Comment by expipiplus1** [comment]
Thanks!
